transitions/wave.h

```diff
@@ -1,6 +1,9 @@
 #ifndef TRANSITIONS_WAVE_H
 #define TRANSITIONS_WAVE_H
 
+// Usage: TrWaveX<COLOR, FADEOUT_MS, WAVE_SIZE, WAVE_MS, WAVE_CENTER>
+// COLOR: COLOR
+// FADEOUT_MS, WAVE_SIZE, WAVE_MS, WAVE_CENTER: FUNCTIONS
```

Comment:
s/FUNCTIONS/FUNCTION/  (here and in other places)

Reply:
sorry, huh?

Reply:
Done.

Reply:
Sorry, s/A/B/ is search-and-replace in ed, sed and vi, which are ancient unix text editors of various kinds.

Reply:
I use find with sed to batch replace text in my config files 😃